contextualize_eval.cpp

```diff
@@ -0,0 +1,88 @@
+#include "contextualize_eval.hpp"
+#include "ast.hpp"
+#include "eval.hpp"
+#include "backtrace.hpp"
+#include "to_string.hpp"
+#include "parser.hpp"
+
+namespace Sass {
+
+  Contextualize_Eval::Contextualize_Eval(Context& ctx, Eval* eval, Env* env, Backtrace* bt)
+  : Contextualize(ctx, env, bt), eval(eval)
+  { }
+
+  Contextualize_Eval::~Contextualize_Eval() { }
+
+  Selector* Contextualize_Eval::fallback_impl(AST_Node* n)
+  { return Contextualize::fallback_impl(n); }
```

Comment:
Sorry I think I'm still confused by class. 

```
Contextualize_Eval::operator()(Selector_List* s) {
  // Is there a difference between these two I'm missing?
  return Contextualize::operator ()(s);
  return s->preform(&instance_of_contextualize);
}
``` 

If so then you can remove this classes operators and fallback to the Contextualize

```c++
Selector* Contextualize_Eval::fallback_impl(AST_Node* n) {
  return n->perform(&instance_of_contextualize);
}
```

Which will hit `Contextualize::fallback_impl` if that operator type isn't defined.

*** 

Is it simply that you wish to opt-out of some `Contextualize` operators? It's unclear from this diff alone.